src/components/FeaturePanel/ImageSection/FeatureImage.tsx

```diff
@@ -7,34 +7,23 @@ import { Feature } from '../../../services/types';
 import { InlineSpinner } from './InlineSpinner';
 import { t } from '../../../services/intl';
 import { nl2br } from '../../utils/nl2br';
+import { Photo } from './Photo';
 
 const Wrapper = styled.div`
   position: relative;
-  background: #ddd url(${({ link }) => link ?? ''}) center center no-repeat;
-  background-size: ${({ portrait }) => (portrait ? 'contain' : 'cover')};
+  background: #ddd
+    linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0) 70%,
+      rgba(0, 0, 0, 0.09) 76%,
+      #5b5b5b
+    );
   height: 238px;
   min-height: 238px; /* otherwise it shrinks b/c of flex*/
   ${({ uncertainImage }) =>
     uncertainImage
       ? 'box-shadow: inset 0 0 100px rgba(255,255,255,0.3); filter: contrast(0.6) brightness(1.2);'
```

Comment:
Another overlay is displayed when we show an image from mappilary (as opposed to images from wikipedia). Because we are not really sure, the mapillary image shows anything useful. I like to fade the image a little with this filter. 
1. it would really need to stay in place for not-panellum images
2. i am not sure if iframe can be overlayed, but i think it can. Again perhaps with pointer-events?